data/human/free worlds side plots.txt

```diff
@@ -11,6 +11,88 @@
 # You should have received a copy of the GNU General Public License along with
 # this program. If not, see <https://www.gnu.org/licenses/>.
 
+
+
+mission "FW Convoy to Zug"
+	name "Convoy to Zug"
+	description "Escort these Freighters to Zug for an important project. Expect pirate raids."
```

Comment:
The freighters are literally Freighters, but I'd still use lower case "freighter," just so that we're not worrying about going back and forth between the two, particularly if we ever decide to change which freighters we're escorting.
```suggestion
	description "Escort a convoy of freighters to <destination> for an important project. Expect pirate raids."
```